CefSharp.Example/Handlers/AudioHandler.cs

```diff
@@ -2,26 +2,32 @@
 //
 // Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
 
-using System;
 using CefSharp.Enums;
 using CefSharp.Structs;
+using System;
+using System.IO;
 
 namespace CefSharp.Example.Handlers
 {
     public class AudioHandler : Handler.AudioHandler
     {
+        private readonly string pathToSaveAudioData;
```

Comment:
Doesn't look like this is actually used and can be removed.

Reply:
Fixed